### PR TITLE
Chore: Script to generate betterer issue summary

### PR DESCRIPTION
**Pull request by joshhunt**
**What is this feature?**

Adds a script to output a summary of betterer issues, divided by codeowners, to help understand what tasks are remaining.

Example usage:

```bash
$ yarn betterer:issues --test="better eslint" -m "Use data-testid for E2E selectors instead
of aria-label" --single-owner --output ~/dev/grafana/issues
```

Example output:

> Hi @grafana/observability-metrics!
> 
> The following files have been marked as having issues regarding `better eslint: Use data-testid for E2E selectors instead of aria-label`.
> 
> There are 10 issues over 6 files:
> 
> - [ ] 1 issue in `public/app/plugins/datasource/prometheus/components/monaco-query-field/MonacoQueryField.tsx` 
> - [ ] 1 issue in `public/app/plugins/datasource/prometheus/configuration/ExemplarSetting.tsx` 
> - [ ] 1 issue in `public/app/plugins/datasource/prometheus/configuration/ExemplarsSettings.tsx` 
> - [ ] 3 issues in `public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilterItem.tsx` 
> - [ ] 1 issue in `public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryEditorSelector.tsx` 
> - [ ] 3 issues in `public/app/plugins/datasource/prometheus/querybuilder/shared/LabelFilterItem.tsx` 

Help output:

```
yarn betterer:issues
Usage: yarn betterer:issues -t [path] -o [path] -b [string]

Options:
  -t, --template      Path to a template to use for each issue. See source
                      bettererIssueTemplate.md for an example
         [string] [required] [default: "./scripts/cli/bettererIssueTemplate.md"]
  -o, --output        Path to directory to save issues to    [string] [required]
  -b, --test          Name of the betterer test to produce the report for
                                                             [string] [required]
  -m, --test-message  Filter issues containing this message             [string]
  -s, --single-owner  Only use first owner for files with multiple owners
                                                      [boolean] [default: false]
      --help          Show help                                        [boolean]

Missing required arguments: output, test
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -302,6 +305,7 @@
     "calculate-size": "1.1.1",
     "centrifuge": "3.1.0",
     "classnames": "2.3.2",
+    "codeowners": "^5.1.1",
```
Comment (ashharrison90) [inline package.json:308]: this should probably just be a `devDep` i think

**Review by ashharrison90 — APPROVED** [review]
looks good! 🙌

related question: wonder whether stuff like this should just live in a github action? feels a bit weird to maintain and include new deps for something which is basically a dev-only script and not part of grafana itself 🤔

File: package.json (hunk 1 above)
Comment (joshhunt) [reply package.json]: fix ta thanks :)

**Comment by joshhunt** [comment]
Yeah, i understand your point, but I think it's fine.

I don't think actions are the right place for arbitary ad-hoc scripts to run...

**Pull request merged**